### ISSUE TITLE
NotImplementedError: change_ordering for Slice is not implemented

### ISSUE THREAD

**Issue by AzazelHD** [apache/tvm#7217]
As the tittle sais, I am getting this error when trying to convert a model.

``NotImplementedError: change_ordering for Slice is not implemented``

**Comment by insop** [comment]
@AzazelHD
It would help others if you add details, such as from what software frame you convert, what type of model you are referring, and so on.

**Comment by AzazelHD** [comment]
Yeah. I got a model that is on pytorch format. I used ``pytorch2keras`` library (python), and posted this error there. They said it must be an ``onnx2keras`` error.

The model was trained on GPU, but on keras I wanna predict since my GPU is AMD. 

Model was on format BCHW, but CPU only admits BHWC format, so I passed ``pytorch_to_keras`` ``change_ordering=True``. This arguments attempts to convert BCHW to BHWC.

But then I got that error
